eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/utils/InitDataLoader.java

```diff
@@ -64,5 +71,21 @@ private void createMeetings() {
             .finishTime(System.currentTimeMillis() + 3600000 * 2)
             .createdAt(System.currentTimeMillis())
             .build());
+        List<Meeting> testMeetings = new ArrayList<>();
+        for(int i = 0; i < 10; i++) {
```

Comment:
Hmm, ebből is lehetne csinálni egy szép lambdát. 
`IntStream`-mel. Akkor, ha map-elsz, nem kell a testMeetings